go/billing/models.py

```diff
@@ -147,6 +149,11 @@ def calculate_credit_cost(cls, message_cost, storage_cost, markup_percent,
         base_cost = message_cost + storage_cost
         if session_created:
             base_cost += session_cost
+        if session_length and session_unit_length and session_unit_cost:
+            units = (
+                Decimal(session_length)/Decimal(session_unit_length)
```

Comment:
Sorry to bikeshed -- pep8 will probably complain about the `/` not having spaces around it.